NotesBackEnd/tests/Notes.Api.Tests/packages.config

```diff
@@ -1,12 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Castle.Core" version="4.2.0" targetFramework="net471" />

```

Comment:
You need Castle? 😮

Reply:
When I try to remove Castle.Core I get an error message - Unable to uninstall "Castle.Core.4.2.0" because NSibstitute.3.1.0 depends on it - so I Think I really need it :)